localgovdrupal/localgov_forms
# Fix: Address lookup validation

## What does this change?
The LocalGov address lookup element's validation process is losing existing form error messages set by other parts of a form.  Some of its own error messages are also getting lost.  This change fixes this.

I have also updated the wording for error messages shown on required fields like Address line 1 and Postcode.  Instead of saying "Postcode field is required", it now says "You must enter the postcode."  Error message wording courtesy of @DavidBAsch 

## How to test
- Create a new Webform.
- Add a set of radio buttons.  Mark it as `required`.
- Add a LocalGov address lookup element.
- Set the visibility condition of the address lookup element dependent on one of the radio buttons.
- Submit the Webform without selecting any of the radio buttons.  At the moment, form submission completes and an error message about the radio button is displayed in the **confirmation** page rather than on the form page where the form elements are.  Once this fix is applied, the form submission fails as expected.
